test/unit_tests/test_interrogate_setup.py

```diff
@@ -153,4 +169,14 @@ def test_interrogate_setup_py(self):
 set(foo_SETUP_PY_MODULE_DIRS "")""", contents)
             os.remove(outfile)
         finally:
-            shutil.rmtree(rootdir)
+            shutil.rmtree(develdir)
+            shutil.rmtree(srcdir)
+            os.chdir(curdir)
+
+
+def _create_empty_files(filenames):
+    for fname in filenames:
+        directory = os.path.dirname(fname)
+        if not os.path.exists(directory):
+            os.makedirs(directory)
```

Comment:
Rather catch the exception than test for existence before since this is still subject to a race.